### ISSUE TITLE
All PublicBL domains in a single text file ?

### ISSUE THREAD

**Issue by origamiofficial** [nabeelxy/DomainBL#1]
Hi there,
I just found this repo when I search for a specific malicious domain "coolchance.net" through Github. What you're doing is really awesome. I would like to use your blacklist in my home [Pi-hole](https://github.com/pi-hole/pi-hole). But the problem is you create every day another file. If you just made another **single text file** in this repo & put all the **PublicBL domains in the text file** will be helpful. Every day my Pi-hole will check for the new contents in the text file & get all the new domains from there.
Thank You

Some more malicious apex domains I found along with the specific domain I mentioned before:
diverttreat.top
h3bb7fin.cn
bottombenevolent.buzz
qoaaa.com
qraaa.com
vuaaa.com
quaaa.com
gvaaa.com
uqaaa.com
vbaa.net
tc-offer.com

**Comment by nabeelxy** [comment]
At the moment, we generate one file per day. They are always named YYYY-MM-DD.csv as it is a daily pipeline. Would you be able to aggregate these files into one copy from your end?

**Comment by origamiofficial** [comment]
Yeah, did it already. Available at: https://github.com/origamiofficial/PiholeBL
